src/Track.js

```diff
@@ -23,6 +23,7 @@ export default class {
     constructor() {
 
         this.name = "Untitled";
+        this.id = 'track'
```

Comment:
oops, yes this this makes sense. I did now make it optional. I think thats more useful than random ids, what do you think?